.github/workflows/publish.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v2-beta
         with:
-          node-version: '14'
+          node-version: '16.10.0'
```

Comment:
How this will ignore require?

Reply:
It will help publish changes :smile: 
https://github.com/energywebfoundation/shared-configs/runs/5247951025?check_suite_focus=true